WalletWasabi.Fluent/ViewModels/Wallets/Send/PrivacySuggestionsFlyoutViewModel.cs

```diff
@@ -46,7 +47,7 @@ public  async Task BuildPrivacySuggestionsAsync(Wallet wallet, TransactionInfo i
 		}
 
 		var suggestions =
-			await ChangeAvoidanceSuggestionViewModel.GenerateSuggestionsAsync(info, destination, wallet, transaction);
+			await ChangeAvoidanceSuggestionViewModel.GenerateSuggestionsAsync(info, destination, wallet, transaction, cancellationToken);
```

Comment:
I'm confused here. L50 computes suggestions (heavy operation) and L52 checks whether we have a change output and if we have, then shows the computed suggestions.

Why do we compute the suggestions to discard them if there is no change input? Or am I missing something?

Reply:
Not my code and logic, but you have a really good point, I will change it.